Resources/public/css/views/tabs/relations.css

```diff
@@ -33,3 +33,24 @@
 .ez-view-locationviewrelationstabview .ez-relations-box-list-no-content {
     font-style: italic;
 }
+
+.ez-view-locationviewrelationstabview .ez-breadcrumbs-list {
+    font-size: 90%;
```

Comment:
I think it's more than time to put that CSS code in a dedicated CSS module as we pretty much the same code in different places. Also, I don't see the need for the 90% font size here, I think this was just copy pasted from the Location view breadcrumb but in this context and in the related content tab, this is weird.